ibis/tests/all/test_window.py

```diff
@@ -7,16 +9,20 @@
     Csv,
     Impala,
     MySQL,
-    OmniSciDB,
     Pandas,
     Parquet,
     PostgreSQL,
-    PySpark,
     Spark,
     SQLite,
 )
 
 
+def _win_count(se):
```

Comment:
Not great to repeat the same function one, can you import the one in the pandas backend, even if it's private.